packages/oncokb-frontend-commons/src/components/RefComponent.tsx

```diff
@@ -11,46 +11,75 @@ export default class RefComponent extends React.Component<{
     componentType: 'tooltip' | 'linkout';
 }> {
     render() {
-        const parts = this.props.content.split(/pmid|nct/i);
+        const parts = this.props.content.split(/pmid|nct|abstract/i);
 
         if (parts.length < 2) {
             return <span>{this.props.content}</span>;
         }
 
-        const ids = parts[1].match(/[0-9]+/g);
-
-        if (!ids) {
-            return <span>{this.props.content}</span>;
-        }
+        // Slicing from index 1 to end to rejoin the abstract title and link
+        // when there's the case that they also contain the string 'Abstract'
+        // Example :
+        //     (Abstract: Fakih et al. Abstract# 3003, ASCO 2019. https://meetinglibrary.asco.org/record/12411/Abstract)
+        const abstractParts = parts
+            .slice(1)
+            .join('Abstract')
+            .split(/(?=http)/i);
+        const isAbstract = !(abstractParts.length < 2);
 
+        let abstract = '';
```

Comment:
`abstract` is a reserved keyword. I am surprised compiler is okay with this.

Reply:
Yeah compiler didn't give any warnings probably because `abstract` is [not reserved anymore](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#reserved_words)?

Also, refactoring the code based on the above comment allowed me to remove the `abstract` and `abstractLink` variables.